### DISCUSSION TITLE
Can we use the new features in v1.11.2 on devnet?

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#7323] [Storage Provider] [answered]
I tried to renew sectors.
`lotus-miner sectors renew --sector-file /tmp/sector 1555200`
and i got this response
> Renewing 1 sectors: 
{
  "Extensions": [
    {
      "Deadline": 21,
      "Partition": 0,
      "Sectors": "46",
      "NewExpiration": 1577324
    }
  ]
}
1 sectors renewed

my sectorfile content is:
> cat /tmp/sector 
46

sector 46 still have old lifecycle. and if i use lotus-miner sectors extend command. i got success.
> 46  Proving  YES      YES     703377 (in 33 weeks 5 days)   2      1.5KiB

and also when i change CommittedCapacitySectorLifetime = "46.656ms" to anytime . seems it will not effect the sectors lifecycle in devnet.

**Comment by coinmini** [comment]
the same here

**Comment by LexLuthr** [comment, marked as answer]
@dayou5168 Are you still running into this problem on devnet?

**Reply by de-authority** [reply]
I think it's fixed
